Musicca/app/src/main/java/com/example/musicca/activities/SongPlaylistActivity.java

```diff
@@ -42,19 +62,29 @@ protected void onCreate(Bundle savedInstanceState) {
         ivSongAlbum = findViewById(R.id.ivSongAlbum);
         tvTitle = findViewById(R.id.tvTitle);
         tvArtist = findViewById(R.id.tvArtist);
-        btnReturnPlaylist = findViewById(R.id.btngotoPlaylist);
+        btnReturnPlaylist = findViewById(R.id.btnReturnPlaylist);
+
+        ivPrevious = (ImageView) findViewById(R.id.ivPrevious);
+        ivPlayPause = (ImageView) findViewById(R.id.ivPlayPause);
```

Comment:
Do these buttons work yet?

Reply:
At this point they do not. They are serving as image placeholders